config/crd/kustomization.yaml

```diff
@@ -12,7 +12,7 @@ resources:
 patchesStrategicMerge:
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
 # patches here are for enabling the conversion webhook for each CRD
-- patches/webhook_in_logpipelines.yaml
+#- patches/webhook_in_logpipelines.yaml
```

Comment:
Could you please explain why we don't need it anymore?

Reply:
The webhook file here is a conversion webhook which was included before by mistake. Also we dont need a conversion webhook right now so I commented it out.